app/models/user.rb

```diff
@@ -85,6 +85,7 @@ class User
   validate :secure_password
 
   TIME_BEFORE_INACTIVE = 2.weeks
+  TIME_BEFORE_DISABLED_BY_FUND = 9.months
```

Comment:
Suggestion/Bike Shedding: move these out of a model into something like config. Not the biggest deal at the end of the day but overall app config can benefit from being in one place rather than spread out.

Reply:
ah, I hear you. We've been doing this like so because of the rails convention to declare global stuff in initializers or `application.rb` and other constants more limited in scope in the module they're used in. Since this is specific to the user obj and not really used outside of here except in a print statement, probably best to follow convention and keep them closer to the place they're used.